test/libsolidity/smtCheckerTests/operators/conditional_assignment_3.sol

```diff
@@ -9,5 +9,7 @@ contract C {
 		assert(c > a);
 	}
 }
+// ====
+// SMTIgnoreCex: yes
```

Comment:
Out of curiosity: What does the `SMTIgnoreCex` flag do?

Reply:
It removes the counter-example trace from the output of the tests. It has been added by @leonardoalt to work around the non-determinism in Spacer (which quite often gives different counter-example on different platforms, when the test allows different CEXs).

Reply:
Ah I see. Cex presumably stands for Counter*ex*ample :)